packages/oui-navbar/src/navbar.html

```diff
@@ -133,6 +140,37 @@ <h3 class="oui-navbar-menu__title" ng-bind="::asideLink.headerTitle || asideLink
                            aria-hidden="true"></i>
                     </button>
                 </li>
+                <li class="oui-navbar-menu__item oui-navbar-notification oui-navbar-notification__placeholder"
+                    ng-if="asideLink.subLinks.length === 0"
+                    oui-navbar-group="{{::asideLink.name}}">
+                    <div class="oui-navbar-notification__placeholder-content">
+                        <i class="oui-icon oui-icon_bicolor oui-icon-success_circle oui-navbar-notification__icon"
+                           aria-hidden="true">
+                        </i>
+                        <div>
+                            <div class="oui-navbar-notification__placeholder-content-title"
+                                 ng-bind="$ctrl.config.translations.notification.noNotification"></div>
+                            <div class="oui-navbar-notification__placeholder-content-description"
+                                 ng-bind="$ctrl.config.translations.notification.noNotificationDescription"></div>
+                        </div>
+
+                    </div>
+                </li>
+                <li class="oui-navbar-menu__item oui-navbar-notification oui-navbar-notification__placeholder"
```

Comment:
`oui-navbar-notification oui-navbar-notification__placeholder`

A block and an element at the same level ?
Should be a modifier instead, `oui-navbar-notification_placeholder` ?

Or maybe a new block:
`<li class="oui-navbar-menu__item oui-navbar-notification-placeholder">`